sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,654 @@
+          `predict_proba`, `decision_function` or `predict` in that order.
+
+    n_jobs : int, default=None
+        The number of jobs to run in parallel for both `fit` the `estimators`.
+        `None` means 1 unless in a `joblib.parallel_backend` context. -1 means
+        using all processors. See Glossary for more details.
+
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by `np.random`. Used to set the `cv`.
+
+    Attributes
+    ----------
+    estimators_ : list of estimator object
+        The base estimators fitted.
+
+    named_estimators_ : Bunch object, a dictionary with attribute access
+        Attribute to access any fitted sub-estimators by name.
+
+    final_estimator_ : estimator object
+        The classifier to stacked the base estimators fitted.
+
+    predict_method_ : list of string
+        The method used by each base estimator.
+
+    References
+    ----------
+    .. [1] Wolpert, David H. "Stacked generalization." Neural networks 5.2
+       (1992): 241-259.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.svm import LinearSVC
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from sklearn.ensemble import StackingClassifier
+    >>> X, y = load_iris(return_X_y=True)
+    >>> estimators = [
+    ...     ('lr', LogisticRegression(solver='lbfgs', multi_class='auto',
+    ...                               tol=1e-1)),
+    ...     ('svr', LinearSVC(tol=1e-1, random_state=42))
+    ... ]
+    >>> clf = StackingClassifier(
+    ...     estimators=estimators,
+    ...     final_estimator=RandomForestClassifier(n_estimators=10,
+    ...                                            random_state=42),
+    ...     cv=5,
+    ... )
+    >>> from sklearn.model_selection import train_test_split
+    >>> X_train, X_test, y_train, y_test = train_test_split(
+    ... X, y, stratify=y, random_state=42
+    ... )
+    >>> clf.fit(X_train, y_train).score(X_test, y_test)
+    0...
+
+    """
+    def __init__(self, estimators, final_estimator=None, cv=None,
+                 predict_method='auto', n_jobs=None, random_state=None,
+                 verbose=0):
+        super().__init__(
+            estimators=estimators,
+            final_estimator=final_estimator,
+            cv=cv,
+            predict_method=predict_method,
+            n_jobs=n_jobs,
+            random_state=random_state,
+            verbose=verbose
+        )
+
+    def _validate_meta_estimator(self):
+        super()._validate_final_estimator(
+            default=LogisticRegression(random_state=self.random_state)
```

Comment:
lbfgs is not randomised I think.

Reply:
true. It was written with default for liblinear